src/modules/notifications/index.js

```diff
@@ -33,7 +36,9 @@ import type {
   Schedule,
 } from './types';
 
-type OnNotification = Notification => any;
+type CompletionHandler = BackgroundFetchResultValue => void;
+
+type OnNotification = (Notification, CompletionHandler) => any;
```

Comment:
Remove `CompletetionHandler` from `OnNotification`. This means we don't need `CompletionHandler` at all in this file.